packages/react/index.ts

```diff
@@ -186,6 +187,8 @@ function failureSummary(
  * @return {Promise}
  */
 function checkAndReport(node: Node, timeout: number): Promise<void> {
+  const disableCache = conf['disableCache'];
```

Comment:
I'm all for this, but would suggest this be called `disableDeduplicate`.

```suggestion
  const disableCache = conf['disableDeduplicate'];
```

Reply:
Yup I can make that change :)